### PR TITLE
fix(brh.data-commons.org): use use data-portal 3.22.1 (04-01-2022)

### PR DESCRIPTION
**Pull request by PlanXCyborg**

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# brh.data-commons.org/manifest.json
## :warning: Services on branch
- hatchery

File: brh.data-commons.org/manifest.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "hatchery": "quay.io/cdis/hatchery:feat_multiple_paymodels",
     "indexd": "quay.io/cdis/indexd:2022.03",
     "metadata": "quay.io/cdis/metadata-service:1.6.4",
-    "portal": "quay.io/cdis/data-portal:feat_multiple_paymodels",
```
Comment (mfshao) [inline brh.data-commons.org/manifest.json:19]: we should keep this branch

I have rebased that branch w/ master so we should be fine

Comment (mfshao) [inline brh.data-commons.org/manifest.json:19]: we should keep this branch

I have rebased that branch w/ master so we should be fine

**Comment by mfshao** [comment]
super outdated PR, no longer needed, closing

**Pull request closed without merging**